### ISSUE TITLE
Feature request: List of subscribed channels

### ISSUE THREAD

**Issue by anubis-genix** [yuliskov/SmartTubeNext#17]
Please add a list of subscribed channels in the subscription menu so we can preselect by channel.

**Comment by steve51184** [comment]
This is one of only 2 missing features stopping me move from Smart YouTube TV

**Comment by zejefferson** [comment]
As soon as this is implemented, I'm switching to this app!

**Comment by jalarab** [comment]
I would like this too.

**Comment by buthig666** [comment]
Me too

**Comment by Ph4te** [comment]
I'm still using Smart Youtube TV, because it has this feature. I would really like this app to have it, too.

**Comment by buthig666** [comment]
> I'm still using Smart Youtube TV, because it has this feature. I would really like this app to have it, too.

This feature has been there for several updates now.

**Comment by Ph4te** [comment]
> > I'm still using Smart Youtube TV, because it has this feature. I would really like this app to have it, too.
> 
> This feature has been there for several updates now.

No, not really. If I go to my subscriptions, I see every new video lumped together. And if I go to channels in the main menu I have to actively select a channel after which a new windows opens which shows the new videos. After that I have to go back to choose another channel.

The way it is implemented in smart YouTube TV is superior. In it you see your channels on the left side and an overview of the videos on the right side and you just have to scroll down to refresh the view on the new videos.

**Comment by buthig666** [comment]
I see what you mean,  you are hoping for an integration of the menu for channels as a filtered search within subscriptions.  I could see that being convenient.

**Comment by Ph4te** [comment]
> I see what you mean, you are hoping for an integration of the menu for channels as a filtered search within subscriptions. I could see that being convenient.

Correct.

**Comment by ahmaddxb** [comment]
Yes I really miss this list we used to have. I regularly miss watching video because of this missing feature. Please implement it in your new app. Thanks

**Comment by albino1** [comment]
After sitting on SmartYouTubeTV for quite a while now I finally decided to break down and try SmartTubeNext and, wow, it's so good!  It's actually better than I could have even imagined, and a lot of the things I thought would be an issue when it was first released seem to have already been solved.  It's so snappy and just great to use, I'm really impressed.

However, I do think that this feature request is the most glaring thing that immediately stood out to me as missing.  I actually do kind of like how the Subscriptions and Channel tabs are setup in SmartTubeNext, and maybe it will grow on me, but some method to just look at a specific subscribed channel and see their latest videos in order like the old app would be incredibly useful.   Just my two cents, thanks!
